int16 eid
int16 mob_type
int16 x
int16 y
int16 z
int16 length=1
int16 width=1
int16 height=1
int16 head_yaw
int16 head_pitch
int16 velocity_x
int16 velocity_y
int16 velocity_z

